docs/src/pages/guides/probes.md

```diff
@@ -89,6 +89,24 @@ probes:
         ping: true
 ```
 
+### Postgres Request
+
+Monika provides a way to check your postgres database's health with a 'postgres' type probe.
+
+```yaml
+probes:
+  - id: 'postgres-01'
+    name: database health
+    description: ensure db health
+    interval: 30 # in seconds
+    postgres:
+      - host: 172.15.0.1
+        port: 5432
+        database: my-database
```

Comment:
Was there any possibility that I could probe my database without having to supply a database name? Just to check the connection, for example

Reply:
It is possible to have no database with this driver.. but the user / password is expected to be string. In the other app we were looking at, database IS mandatory, but defaults to a 'template1' if not specified.